app/components/candidate_interface/rejection_reasons_component.rb

```diff
@@ -10,10 +10,15 @@ def initialize(application_form:)
     def application_choice_rows(application_choice)
       [
         course_details_row(application_choice),
+        status_row(application_choice),
         rejection_reasons_row(application_choice),
       ].compact
     end
 
+    def render?
```

Comment:
Is this being called anywhere?

Reply:
Not by our code but it's called by the `view_components` framework. It's just an easy way to do conditional rendering.

Reply:
Oh right okay, we explicitly call it in a few other components irrc. Thanks for the info